### DISCUSSION TITLE
5.7.0 Release

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16590] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.7.0'>5.7.0 Release</a>.</em>

**Comment by s-ohnishi** [comment]
@niden I installed Phalcon5.7.0 with PECL, but the result of `phpinfo()` is 5.7.0, but it says that it has been fixed `Phalcon\Db\Column::TYPE_BINARY`,`Phalcon\Db\Column::TYPE_TINYINTEGER ` remained the same value.
Is there something wrong with the PECL file?

**Reply by niden** [reply]
I remember seeing the commit that corrected this but now looking at the code it seems the merge reverted it back. Ugh.

Nice catch. I will issue a correction shortly.

**Reply by niden** [reply]
I pushed the commit and tests on the 5.0 branch. Thanks again

**Reply by s-ohnishi** [reply]
It's not reflected yet.
`php -r "echo (new \Phalcon\Support\Version())->get();"` returns `5.6.2` and `php -r "echo Phalcon\Db\Column::TYPE_BINARY, ',',Phalcon\Db\Column::TYPE_TINYINTEGER;"` returns `26,26`.
Will it take more time?

**Reply by niden** [reply]
The code is in the 5.0.x branch. I corrected the version also (which I forgot during the deployment) so this does report 27, 26

If you want this right now, compile it from 5.0.x, otherwise I will get this out in the next release. Sorry about that.

**Reply by s-ohnishi** [reply]
I don't have an environment to compile Phalcon from source, so I'll wait for the next release and install it using PECL.
By then, I hope that as many pending cases as possible will be completed.
